Mathlib/NumberTheory/Padics/PadicNumbers.lean

```diff
@@ -124,7 +124,6 @@ theorem norm_zero_iff (f : PadicSeq p) : f.norm = 0 ↔ f ≈ 0 := by
     by_contra hf
     unfold norm at h
     split_ifs at h
-    · contradiction
```

Comment:
The proof now matches the mathlib3 version more closely:
https://github.com/leanprover-community/mathlib/blob/65a1391a0106c9204fe45bc73a039f056558cb83/src/number_theory/padics/padic_numbers.lean#L118-L119